operator/src/main/java/oracle/kubernetes/operator/helpers/CRDHelper.java

```diff
@@ -112,6 +119,20 @@ static V1beta1CustomResourceDefinitionSpec createSpec(KubernetesVersion version)
       return spec;
     }
 
+    static List<V1beta1CustomResourceDefinitionVersion> getCRDVersions() {
+      List<V1beta1CustomResourceDefinitionVersion> versions =
+          Arrays.stream(KubernetesConstants.DOMAIN_ALTERNATE_VERSIONS)
+              .map(e -> new V1beta1CustomResourceDefinitionVersion().name(e).served(true))
+              .collect(Collectors.toList());
+      versions.add(
```

Comment:
i assume this results in the right order overall...

Reply:
Yes, correct.  For k8s 1.10 - 1.12, I found that the version field had to be set to the first entry in versions.  K8s 1.13 removed that restriction.